cobl2/datatables.py

```diff
@@ -42,6 +44,13 @@ def col_defs(self):
         ]
 
 
+class CoblGlottologCol(Col):
+    def format(self, item):
+        if item.glottocode:
+            return '<a href="%s">%s</a>' % (url(item.glottocode), item.glottocode)
```

Comment:
Preferably, HTML should be constructed using the `clld.web.util.htmllib.HTML` object, i.e. as
```python
HTML.a(item.glottocode, href=url(item.glottocode))
```
to ensure proper escaping (even though this shouldn't be a problem here).